src/qt/sendcoinsdialog.cpp

```diff
@@ -753,10 +763,7 @@ void SendCoinsDialog::coinControlFeatureChanged(bool checked)
 // Coin Control: button inputs -> show actual coin control dialog
 void SendCoinsDialog::coinControlButtonClicked()
 {
-    CoinControlDialog dlg(platformStyle);
-    dlg.setModel(model);
-    dlg.exec();
-    coinControlUpdateLabels();
+    coinControlDialog()->open();
```

Comment:
Notes for other reviewers from [Qt docs](https://doc.qt.io/qt-5/qdialog.html#exec):
> Avoid using this [`exec()`] function; instead, use `open()`.